index.html

```diff
@@ -55,10 +55,21 @@
 
 			<div id="canvas-div"></div>
 			<div id="palette-div"></div>
+			<div id="gameboy-palette-div"></div>
 			<label id="selection-explanation-text">
 				How to Copy <img class="imgInSelectionExplanation" src="img/selectionBright.png"></img> Selection: <b>hold down</b> <span>alt</span> and <b>drag</b> the selected area.
 			</label>
 			<div id="cursor-coords-display"></div>
+			<div id="palette-select">
+				<label class="container">NES
+					<input type="radio" onclick="selectPalette()" name="radio" id="radioNES">
+					<span class="checkmark"></span>
+			  	</label><br>
+			  	<label class="container">GB
```

Comment:
I would spell this out as GameBoy ... better accessibility for folks to see the full name

Reply:
I agree. Although I may have to adjust the text size as the word does an odd wrap-around effect, so the button is on one line with the text underneath it.

Reply:
SGTM. Lowering the font-size a tad bit is okay with me, just so long that all labels are affected in the same way and that we move this to a classname. :)